android/app/src/flavors/auto-enabled/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@arnaudvergnet could you please help us with understanding where to put Android Manifest and xml files for mix-ins like androidauto-enabled, gms-enabled, etc?

Reply:
manifest.srcFile can have only one value, unlile java.srcFile**s**

It looks like that we need to add a new sourceSet in build.gradle for androidAutoEnabled case.

Reply:
Should be better to move everything in a new module, this way it handles merging manifests I think

Reply:
The idea of a new module didn't fly, unfortunately.